ghost/members-api/lib/controllers/MemberController.js

```diff
@@ -1,4 +1,9 @@
 const errors = require('@tryghost/errors');
+const tpl = require('@tryghost/tpl');
+
+const messages = {
+    blockedEmailDomain: 'Signups from this email domain are currently restricted.'
```

Comment:
Would rename the error message to e.g. `Memberships from this email domain are currently restricted.`